Makefile

```diff
@@ -0,0 +1,112 @@
+SHELL = /bin/bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DELETE_ON_ERROR:
+MAKEFLAGS += --warn-undefined-variables
+MAKEFLAGS += --no-builtin-rules
```

Comment:
The values look reasonable to me. Do you have a specific source of inspiration for them?

Reply:
Takne from https://tech.davis-hansson.com/p/make/. It has a few other things that I don't think are helpful as e.g. they need a newer make version than available in MacOS Catalina by default.